src/service/game_service.py

```diff
@@ -53,7 +53,11 @@ def close(id: str) -> str:
         target_game: Game = GamesDAO.retrieve(id)
         if not target_game: raise ResourceNotFound
 
-        champion_name: str = max(target_game.teams, key=attrgetter('goals')).name
+        if target_game.game_is_tie():
+          champion_name = 'Empate'
+          
+        else:
+          champion_name: str = max(target_game.teams, key=attrgetter('goals')).name
```

Comment:
In the backend we will never specify such things as explicit (*hardcoded*) texts that will be displayed to the final user. It's running backwards in terms of decoupling and maintainability strategies.

A little more simple way of showing problems with this approach is that this is the only portuguese word in a 600+ lines codebase.